Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```
@@ -1234,12 +1233,95 @@ script:
+    description: get papi property info without default rule. to get default rule, please use "get-papi-property-rule" command.
+    name: akamai-get-papi-property-byname
+  - arguments:
+    - description: Unique identifier for the contract
+      name: contract_id
+      required: true
+    - description: Unique identifier for the group
+      name: group_id
+      required: true
+    - defaultValue: PapiProperty.ByGroup
+      description: Custom output context path, default is "PapiProperty.ByGroup"
+      name: context_path
+    description: 'Lists properties available for the current contract and group. '
+    name: akamai-list-papi-property-bygroup
+  - arguments:
+    - description: Unique identifier of the contract
+      name: contract_id
+      required: true
+    - description: Unique identifier for the group
+      name: group_id
+      required: true
+    - description: Unique identifier of the property
+      name: property_id
+      required: true
+    description: get papi property info by id without default rule. to get default rule, please use "get-papi-property-rule" command.
+    name: akamai-get-papi-property-byid
+  - arguments:
+    - description: Unique identifier for the contract.
+      name: contract_id
+      required: true
+    - description: Unique identifier for the property.
+      name: property_id
+      required: true
+    - description: Unique identifier for the group.
+      name: group_id
+      required: true
+    - description: The property version on which to base the new version.
+      name: create_from_version
+      required: true
+    description: 'Create a new property version based on any previous version.          All data from the createFromVersion populates the new version, including its rules and hostnames. '
+    name: akamai-new-papi-proerty-version
+  - arguments:
+    - description: Unique identifier for the contract.
+      name: contract_id
+      required: true
+    - description: Unique identifier for the group.
+      name: group_id
+      required: true
+    - description: Unique identifier for the property.
+      name: property_id
+      required: true
+    description: This lists all activations for all versions of a property, on both production and staging networks.
+    name: akamai-list-papi-proerty-activations
+  - arguments:
+    - description: Unique identifier for the contract.
+      name: config_id
+      required: true
+    description: '        Lists the activation history for a configuration.          The history is an array in descending order of submitDate.          The most recent submitted activation lists first. Products: All.'
+    name: akamai-list-appsec-configuration-activation-history
+  - arguments:
+    - description: Filter the results by cnameFrom. Supports wildcard matches with *.
+      name: hostname
+      required: true
+    - auto: PREDEFINED
+      description: Network of activated hostnames, either STAGING or PRODUCTION. Or leave it BLANK
```

```suggestion
      description: Network of activated hostnames, either STAGING or PRODUCTION. Or leave it blank.
```
If left blank, does it include both staging and production?